base_wamas_ubl/lib/wamas/wamas_grammar/ausk.py

```diff
@@ -0,0 +1,739 @@
+from collections import OrderedDict
+
+grammar = OrderedDict(
+    {
+        "Telheader_Quelle": {
+            "type": "str",
+            "length": 10,
+            "dp": False,
+            "ubl_path": False,
+            "df_val": "SYSLOG",
```

Comment:
@nilshamerlinck This should either be hardcoded to ODOO or being configurable somewhere. It's the name of the data source inside WAMAS.